fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1401,12 +1408,19 @@ ACTOR void setupAndRun(std::string dataFolder, const char *testFile, bool reboot
 	try {
 		//systemActors.push_back( startSystemMonitor(dataFolder) );
 		if (rebooting) {
-			wait( timeoutError( restartSimulatedSystem( &systemActors, dataFolder, &testerCount, &connFile, &startingConfiguration, tlsOptions, extraDB), 100.0 ) );
+			wait( timeoutError( restartSimulatedSystem( &systemActors, dataFolder, &testerCount, &connFile, &startingConfiguration, tlsOptions, extraDB, whiteListBinPaths), 100.0 ) );
+			// FIXME: snapshot restore does not support multi-region restore, hence restore it as single region always
+			if (restoring) {
+				std::string config =  "usable_regions=1";
+				startingConfiguration = makeString(config.size());
+				uint8_t* ptr = mutateString(startingConfiguration);
+				memcpy(ptr, ((uint8_t*)config.c_str()), config.size());
```

Comment:
You can replace all of this with `startingConfiguration = LiteralStringRef("usable_regions=1");`